release_notes/topics/product/7_4_final.adoc

```diff
@@ -162,7 +162,7 @@ More than 1100 issues were fixed during this release. For details on the fixed i
 
 = Known issues
 
-This release includes the following link:https://issues.redhat.com/issues/?jql=project%20%3D%20KEYCLOAK%20AND%20issuetype%20%3D%20Bug%20AND%20fixVersion%20%3D%209.0.4%20AND%20component%20not%20in%20(Infra%2C%20Testsuite%2C%20%22Testsuite%20-%20Performance%22)[known issues] and in particular these higher level issues.
+This release includes the following link:https://issues.redhat.com/browse/KEYCLOAK-13929?filter=12357686[known issues] and in particular these higher level issues.
```

Comment:
This filter is different to the original and unrelated to the known issues, but at the same time, the original does not work either since the components and versions have changed. I suggest removing the link completely